.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "arrowParens": "always",
+  "arrowParens": "always",  // to wrap implicit return values in parentheses?
```

Comment:
```
const exampleToLower = str => str.lowerCase();

const exampleConcat = (a,b) => a+ b();
```

This rule is to enfore the parentheses around the function args, it's a stylistic choice but I find that it increases readability and adds consistency. Any arrow function with more than one arg requires the parentheses, this rule enforces is for arrow functions with only a single arg as well

Reply:
ah, yes. i'm into it.